tox.ini

```
@@ -1,11 +1,11 @@
 [tox]
 envlist =
-    py37,py38,py39,py310,py311
+    py39,py310,py311,py312,py313
 
 [testenv]
 usedevelop = true
 deps =
     pytest-cov
     responses
```

this suggestion slipped my mind for #108, which could be done here for convenience.
```suggestion
    numpy
```
(`responses` was there since b225255dcd78e2eb1859922c88d7ed1ffb0bea88 but doesn't seem needed)